examples/sol-contract/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "example-sol-contract"
+version = "0.1.0"
+authors = ["Pyth Data Foundation"]
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "lib"]
+
+[dependencies]
+borsh = "0.9"
+arrayref = "0.3.6"
+pyth-sdk-solana = "0.5.0"
```

Comment:
Can you use local dependency with version here?

```
pyth-sdk-solana = { path = "../", version = "0.6.1" }
```

Reply:
Of course and I found the correct path is: `pyth-sdk-solana = { path = "../../pyth-sdk-solana", version = "0.6.1" }`